client/src/WithdrawBid.js

```diff
@@ -0,0 +1,14 @@
+const WithdrawBid = ({ bid, handleStatusChange }) => {
```

Comment:
The `handleStatusChange` should also be inside the component.

Reply:
I have removed withdraw button because its not in the requirments.